.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,46 @@
+<!--
+For high severity issues, please contact AWS Support directly.
+
+Please provide the following information:
+-->
+
+
+### Summary
+<!-- Please provide a brief outline of the issue -->
+
+
+### Description
+<!-- Provide detailed information about this issue -->
+
+
+
+<!--
+Please use these blocks to comment out appropriate sections
```

Comment:
I didn't understand this line at first. It may be clearer to put notes in sections we don't need.

&lt;!-- Not required for feature requests --&gt;